scripts/genclient.sh

```diff
@@ -38,10 +40,14 @@ then
             fi
             ;;
         o)
+                CLIENT_PATH="$(createConfig $2)"
```

Comment:
Here and for all the other cases I moved the `createConfig` call to the case and passed in the second/third optional argument which is mean to be the cert name.